std/algorithm/mutation.d

```diff
@@ -1539,6 +1555,14 @@ calls to $(D range.popFront).)  $(LI Otherwise, elements are moved
 incrementally towards the front of $(D range); a given element is never
 moved several times, but more elements are moved than in the previous
 cases.))
+
+Params:
+    s = a SwapStrategy
```

Comment:
I think it's clear from the declaration itself that `s` is a `SwapStrategy`. The question is, what does that mean in the context of this function?